test/testscripts/stat_quantile_bars.jl

```diff
@@ -0,0 +1,16 @@
+using Gadfly, RDatasets, Distributions
+
+set_default_plot_size(6inch, 3inch)
+
+plot(dataset("ggplot2", "diamonds"), x=:Price, Stat.quantile_bars, Geom.segment)
+
+dist = MixtureModel(Normal, [(0.5, 0.2), (1, 0.1)])
+xs = rand(dist, 10^5)
```

Comment:
I thought I said don't use random data (maybe ambiguous 😃) in the test . You could e.g. use `dataset("datasets","faithful")`